.editorconfig

```diff
@@ -0,0 +1,20 @@
+# https://EditorConfig.org
+
+root = true
+
+[*]
+indent_size = 4
+indent_style = space
```

Comment:
For .go
> Indentation
> We use tabs for indentation and gofmt emits them by default. Use spaces only if you must.

https://go.dev/doc/effective_go#formatting

Reply:
did not understand the linked PR

but yes, go files and Makefile both accept tabs and spaces (go example of spaces: the headers copyright), so added them to list of  `indent_style=unset`. `go fmt` (run by golangci-lint) will take care of this for us in go files

Reply:
> [The tab character is a very important part of the syntax of Makefiles.](https://stackoverflow.com/a/2131227/4590470)

So Makefile should not be indented using spaces. Yes unset sounds good.

Spaces can exists in go and makefile but not as an indent method for non-comments.